### PR TITLE
Fix the output

### PR DESCRIPTION
**Pull request by Nosferatul**

### CODE REVIEW

**Comment by Nosferatul** [comment]
@jayvdb i think this is a fix

**Comment by JohnMoutafis** [comment]
@Nosferatul you have to remake your commit message. Have a look here on how to structure your commits: http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html

**Comment by Adrianzatreanu** [comment]
Yeah i agree make it: CLI: fix output or so

File: tests/output/ConsoleInteractionTest.py [hunk 1]
```diff
@@ -503,12 +503,13 @@ def test_print_results_for_file(self):
                 {abspath('filename'): ['test line\n', 'line 2\n', 'line 3\n']},
                 {},
                 self.console_printer)
-            self.assertEqual("""
+            self.assertEqual("""[    ]2{1}
```
Comment (Adrianzatreanu) [inline tests/output/ConsoleInteractionTest.py]: What exactly do the numbers mean? Its not that intuitive to me

**Comment by yukiisbored** [comment]
Can you give us a screenshot of coala in action? It's a good idea for this patch so people can review and comment on the new CLI.

File: tests/output/ConsoleInteractionTest.py (hunk 1 above)
Comment (Nosferatul) [inline tests/output/ConsoleInteractionTest.py]: line number where is the problem

**Comment by Nosferatul** [comment]
http://imgur.com/xhWBWb8

**Comment by Nosferatul** [comment]
http://imgur.com/CasQH7E

**Comment by Adrianzatreanu** [comment]
so we will have them numbered but would still accept only letters? think we should accept both numbers/letters if we choose to show numbers too..

**Comment by yukiisbored** [comment]
@Adrianzatreanu That's the plan to make sure the ones have muscle memory of using numbers can get a good experience without adding a learning curve.

**Comment by yukiisbored** [comment]
@Nosferatul That looks nice, is there any a way to add colors to the letters that does the action? So it'll be easier to notice on first glance.

**Comment by yukiisbored** [comment]
Also, I think it'll be nice to have the same warning prefix on the `Here's what you can do` and the options for consistency sake.

**Comment by Nosferatul** [comment]
Ok, I'll try somehow to add colors for letters and add warning prefix. @Adrianzatreanu it accept numbers too as in the screenshot.

**Comment by Nosferatul** [comment]
I can't add colors in docs, the only way i see it is that I create another string with the name of the action and replace the letter with the letter colored everytime `choose_action` is applied.

**Comment by yukiisbored** [comment]
@Nosferatul I'm not expecting being able to add it to the docstring either :P. Anyway, you should do it when you're printing it, feel free to add colorama since I think it only installs colorama when using Windows.

**Comment by yukiisbored** [comment]
Actually, no not colorama. Just use pygments.

**Comment by yukiisbored** [comment]
Okay, I noticed PyPrint does not support having multiple colors in one print? https://gitlab.com/coala/PyPrint/blob/master/pyprint/ConsolePrinter.py

**Comment by sils** [comment]
@Nosferatul you kinda know that "Fix output" isn't particularly specific? ^^

**Comment by Nosferatul** [comment]
http://imgur.com/GTaFXEY

**Comment by yukiisbored** [comment]
@Nosferatul Looks gorgeous! Please fix the tests and we'll get this merged and ready ASAP :)

File: coalib/output/ConsoleInteraction.py [hunk 2]
```diff
@@ -86,6 +87,25 @@ def highlight_text(no_color, text, lexer=TextLexer(), style=None):
...
+    y = -1
+    letter = ''
+    for i, l in enumerate(line, 0):
+        if line[i] == '(':
+            x = i
+        if line[i] == ')':
+            y = i
+        if l.isupper() and x != -1:
+            letter = l
+    firstpart = line[:x+1]
```
Comment (Adrianzatreanu) [inline coalib/output/ConsoleInteraction.py]: maybe case it: first_part? :D

**Comment by Adrianzatreanu** [comment]
just fix the casing and rest looks great to me :) ill ack it, ping me on gitter when youve rebased it and fixed the casing and tests

**Comment by yukiisbored** [comment]
ack https://github.com/coala/coala/pull/4420/commits/0eb0be9e7bf4765dd829d0aeb5a6918fb00d05e1

**Comment by yukiisbored** [comment]
@rultor merge

**Pull request merged**